### PR TITLE
fix: change manually debug url regexp to lib url-regex-safe

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

> This part of the regular expression may cause exponential backtracking on strings starting with '0' and containing many repetitions of '0'.

This issue originated from https://github.com/apache/apisix-dashboard/pull/1417#issuecomment-772585704, which caused The performance on the UI : when type `000` to debug url, it will passed the reg test

mmexport1612678593730

- How to fix?

use [url-regex-safe](https://github.com/niftylettuce/url-regex-safe) instead of manually debug url regexp to test the debug url

___
### Please add the corresponding test cases if necessary.

add test cases to test `000` and empty url

### CODE REVIEW

File: web/cypress/integration/route/online-debug.spec.js [hunk 1]
```diff
@@ -62,4 +65,28 @@ context('Online debug', () => {
       cy.contains(routeLocaleUS['page.route.input.placeholder.requestUrl']).should('not.exist');
     });
   });
+
+  it('shoule not show the invalid url notification', function () {
```
Comment (juzhiyuan) [inline web/cypress/integration/route/online-debug.spec.js:69]: cc @guoqqqi to review.

**Review by guoqqqi — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]
